libs/core/src/components/pds-text/pds-text.scss

```diff
@@ -81,29 +76,15 @@ h1, h2, h3, h4, h5, h6, p, code, pre, em, strong {
   font-style: italic;
 }
 
-
 // Colors/Variants/Sentiment
-$type-colors: (
-  primary: var(--pine-color-text-primary),
-  secondary: var(--pine-color-text-secondary),
-  neutral: var(--pine-color-text-neutral),
-  accent: var(--pine-color-text-accent),
-  danger: var(--pine-color-text-danger),
-  info: var(--pine-color-text-info),
-  success: var(--pine-color-text-success),
-  warning: var(--pine-color-text-warning),
-);
 
-@mixin generate-type-colors($type-colors) {
-  @each $key, $value in $type-colors {
-    .pds-text--color-#{$key} {
-      color: $value;
-    }
-  }
+// Variable is set in TSX
+// If color prop value isn't valid, will default to body text color
+.pds-text {
```

Comment:
Not a showstopper, but could we just default the prop to `primary` and reuse that styling?

Reply:
Do you mean here in the CSS or in the tsx file?

Reply:
The tsx file.